MultiInterface.jl/src/MultiInterface.jl

```diff
@@ -164,26 +102,26 @@ end
 
 function interface_macro(mod, source, interface_exp)
     f, args, R, W = interface_capture(interface_exp)
-    args = add_missing_argnames(args)
```

Comment:
Are you sure you do not want to catch and handle this case?  e.g. If a user defines a method with an argument like `::Foo`?